_includes/pbc/all/integrate-features/202204.0/integrate-glue-api/integrate-the-product-configuration-glue-api.md

```diff
@@ -860,7 +866,7 @@ class WishlistsRestApiDependencyProvider extends SprykerWishlistsRestApiDependen
 {% info_block warningBox "Verification" %}
 
 Ensure that wishlist item CRUD operations support configurable products.
-For an example, see the following response to the `POST https://glue.mysprykershop.com/wishlists/{{wishlist_id}}/wishlist-items` request:
+For an example, see the following response to the `POST https://glue.mysprykershop.com/wishlists/63b14493-021f-59c2-ae70-94041beb5c06/wishlist-items` request:
```

Comment:
why do we specify exact uuid?
shouldn't we use `{{wishlist_uuid}}` instead?

Reply:
It's needed for consistency (`https://glue.mysprykershop.com/carts/...`)